index.bs

```diff
@@ -4365,6 +4366,48 @@ navigated</dfn> steps given |navigable| and |navigation status|:
 
 </div>
 
+#### The browsingContext.historyUpdated Event #### {#event-browsingContext-historyUpdated}
+
+<dl>
+   <dt>Event Type</dt>
+   <dd>
+      <pre class="cddl local-cddl">
+        browsingContext.HistoryUpdated = (
+          method: "browsingContext.historyUpdated",
+          params: browsingContext.HistoryUpdatedParameters
+        )
+
+        browsingContext.HistoryUpdatedParameters = {
```

Comment:
I wonder if we should also include a string indicating the type of the history update to distinguish `pushState` from `replaceState`?

Reply:
I think we could add it eventually. So far we didn't expose it and didn't have a use case for it so adding it would involve extra implementation work. Also we could add the serialized history state to be able to distinguish multiple updates if there are use cases.